File: C++/Mathematical/Armstrong.cpp
#include <iostream>
#include <cmath>
using namespace std;
int main(){
    int num,sum=0,temp;
    cout<<"Enter Number : ";
    cin>>num;
    temp=num;
    while(num){
        sum+=pow(num%10,3);
        num=num/10;
    }
    if(sum==temp)
        cout<<"Number is Armstrong Number";
    else
        cout<<"Number is not Armstrong Number";
    return 0;
}

File: Python/Simple Programs/Mathematical/Armstrong.py
num=int(input("Enter Number to Check : "))
numa=num%10
numb=int(((num%100)-numa)/10)
numc=int(((num%1000)-(numa+numb))/100)
if (num==(numa*numa*numa)+(numb*numb*numb)+(numc*numc*numc)):
    print("Number is an armstrong Number")
else:
    print("Number is not armstrong Number")
